code/public/assets/heart.svg

```diff
@@ -0,0 +1 @@
+<svg id="icon" height="512" viewBox="0 0 128 128" width="512" xmlns="http://www.w3.org/2000/svg"><path d="m105 46.99c0 13.45-9.16 29.21-22.95 42.03a100.5 100.5 0 0 1 -18.05 13.37c-22.28-13.16-41-36.53-41-55.4a21.374 21.374 0 0 1 40.31-9.92q.375.7.69 1.44a21.851 21.851 0 0 1 2.15-3.84 21.378 21.378 0 0 1 38.85 12.32z" fill="#da364c"/><path d="m99.793 54c-.682 11.241-7.436 23.9-17.743 35.02a100.5 100.5 0 0 1 -18.05 13.37c-22.28-13.16-41-36.53-41-55.4a21.263 21.263 0 0 1 3.9-12.32 21.3 21.3 0 0 1 12.32-3.91c8.8 0 17.3 6.525 20.572 14.115a19.466 19.466 0 0 1 4.208-6.365c.25.47-.21-.49 0 0l.018-.041a11.582 11.582 0 0 1 6.017-5.984 21.345 21.345 0 0 1 8.641-1.725c12.446.111 21.871 10.816 21.117 23.24z" fill="#ef5261"/></svg>
```

Comment:
After som googling I get what all of this means (isch) ^^
It's kind of neat, but it looks butt-ugly and it's hard to read.
Consider chopping the code up as the following

`<svg 
id="icon" 
height="512" 
viewBox="0 0 128 128" 
width="512"
xmlns="http://www.w3.org/2000/svg">
<path d="m105 46.99c0 13.45-9.16 29.21-22.95 42.03a100.5 100.5 0 0 1 -18.05 13.37c-22.28-13.16-41-36.53-41-55.4a21.374 21.374 0 0 1 40.31-9.92q.375.7.69 1.44a21.851 21.851 0 0 1 2.15-3.84 21.378 21.378 0 0 1 38.85 12.32z" fill="#da364c"/><path d="m99.793 54c-.682 11.241-7.436 23.9-17.743 35.02a100.5 100.5 0 0 1 -18.05 13.37c-22.28-13.16-41-36.53-41-55.4a21.263 21.263 0 0 1 3.9-12.32 21.3 21.3 0 0 1 12.32-3.91c8.8 0 17.3 6.525 20.572 14.115a19.466 19.466 0 0 1 4.208-6.365c.25.47-.21-.49 0 0l.018-.041a11.582 11.582 0 0 1 6.017-5.984 21.345 21.345 0 0 1 8.641-1.725c12.446.111 21.871 10.816 21.117 23.24z" fill="#ef5261"/>
</svg>`

or something else (well, I see now posting my comment that it did not format as I expected. Point was having each property on a new row)
Also, can you style it in a CSS file instead so you don't need to have the height and width in the component?